### PR TITLE
Add `center` property

### PR DESCRIPTION
**Pull request by blasten**
* Add `center` property. This is required for https://github.com/PolymerElements/paper-icon-button/issues/11
* Fix tests and add a new test for the `center` property
* Update readme.

cc @cdata @notwaldorf @morethanreal

### CODE REVIEW

File: paper-ripple.html [hunk 1]
```diff
@@ -378,17 +386,27 @@
       downAction: function(event) {
         this.resetInteractionState();
         this.mouseDownStart = Utility.now();
+        var xCenter = this.containerMetrics.width / 2;
+        var yCenter = this.containerMetrics.height / 2;
```
Comment (cdata) [inline paper-ripple.html]: Stylistically, it is nice for `var` declarations to appear as close as is reasonable to where they are hoisted.

**Comment by cdata** [comment]
Minor style issue, otherwise LGTM :+1:

File: paper-ripple.html [hunk 2]
```diff
@@ -64,11 +70,9 @@
 
 @event transitionend
 @param {Object} detail
-@param {Object} detail.node The animated node
+@param {Object} detail node The animated node
```
Comment (notwaldorf) [inline paper-ripple.html]: isn't the `.` still valid?

Comment (blasten) [inline paper-ripple.html]: that's right ;)

File: paper-ripple.html (hunk 1 above)
Comment (blasten) [inline paper-ripple.html]: that's right. I think we should split up this function in the next iteration.

File: test/paper-ripple.html [hunk 3]
```diff
@@ -87,31 +97,61 @@
...
           ripple.downAction(mouseEvent);
 
           waveContainerElement = ripple.ripples[0].waveContainer;
-          waveTranslateString = waveContainerElement.style.transform;
 
           ripple.upAction(mouseEvent);
 
           window.requestAnimationFrame(function () {
+            var currentTransform = waveContainerElement.style.transform;
+            var centereD
```
Comment (notwaldorf) [inline test/paper-ripple.html]: i think this `var centereD` is a typo?

Comment (blasten) [inline test/paper-ripple.html]: oh thanks. I fixed this https://github.com/PolymerElements/paper-ripple/blob/master/test/paper-ripple.html#L122

**Pull request merged**